### PR TITLE
More ux qa updates

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- Remove the number of deploys in charm and bundle details, as well as in search
- Pressing slash key should trigger search
- There should be an opaque layer on the visible canvas when the midpoint is activated and when the deployment summary is open
- You should be able to click away from the white layer to go back to the canvas
- You shouldn't be able to go from one deep inspector page to another service

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3756/
Test PASSed.

File: jujugui/static/gui/src/app/components/panel/_panel.scss [hunk 1]
```diff
@@ -11,39 +11,22 @@
...
-    top: $navbar-height;
     right: 0;
-    border-radius: none;
-}
-
-.panel-component.white-box {
+    top: 0;
     bottom: 0;
     left: 0;
     background-color: rgba(255, 255, 255, 0.3);
```
Comment (anthonydillon) [inline jujugui/static/gui/src/app/components/panel/_panel.scss]: The 'overlay' is 70% of #f7f7f7.
```css
background-color: rgba(247, 247, 247, .7);
```

**Comment by anthonydillon** [comment]
Great stuff @huwshimi :+1: with one small comment about the overlay background color.

**Comment by hatched** [comment]
Nice improvements! :+1: QA :ok:

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3782/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3784/
Test PASSed.

**Comment by huwshimi** [comment]
Thanks @anthonydillon  and @hatched. :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**